.travis.yml

```diff
@@ -1,25 +1,31 @@
 language: c
-cache:
-    - apt
-    - ccache
-env:
- global:
-    - FLAVOR=linux_ubuntu_12_64
-before_install:
-    # install build-dependencies as the doc says
-    - sudo sh src/other_builds/OS_notes.$FLAVOR
-    # install packages needed for testing
-    - sudo apt-get install xvfb xauth libgl1-mesa-dri mesa-utils tcsh libjpeg-progs
+
+addons:
+  apt:
+    packages:
+      - f2c
+      - freeglut3-dev
+      - libf2c2-dev
+      - libglew-dev
+      - libglw1-mesa-dev
+      - libinsighttoolkit4-dev
+      - libmotif-dev
+      - libnifti-dev
+      - libxp-dev
+      - libgsl0-dev
+
 install:
-    - cd src
-    - cp other_builds/Makefile.$FLAVOR Makefile
-    - make vastness
+    - F2C_PKG='libf2c2'
+    - F2C_MOD='-dev'
+    - F2C_VERSION='20130926-2_amd64'
+
+    - wget "http://archive.ubuntu.com/ubuntu/pool/universe/libf/${F2C_PKG}/${F2C_PKG}_${F2C_VERSION}.deb"
+           "http://archive.ubuntu.com/ubuntu/pool/universe/libf/${F2C_PKG}/${F2C_PKG}${F2C_MOD}_${F2C_VERSION}.deb"
+    - sudo dpkg -i "${F2C_PKG}${F2C_MOD}_${F2C_VERSION}.deb"
+                   "${F2C_PKG}_${F2C_VERSION}.deb"
```

Comment:
why that particular version of the package? worthwhile adding a comment right here in the travis.yml

Reply:
I haven't been focussing on the travis file recently. f2c is not needed for the build though so I would remove this line. I've changed f2cdir to "converted_from_fortran" to indicate this. I think it might have been causing confusion. It certainly confused me lots. That has unfortunately resulted in many changes in .c files. I shall extract these changes into a single commit for clarity